receiver/varnishreceiver/scraper.go

```diff
@@ -41,7 +41,7 @@ func newVarnishScraper(settings receiver.CreateSettings, config *Config) *varnis
 	return &varnishScraper{
 		telemetrySettings: settings.TelemetrySettings,
 		config:            config,
-		mb:                metadata.NewMetricsBuilder(metadata.DefaultMetricsSettings(), settings),
+		mb:                metadata.NewMetricsBuilder(metadata.DefaultMetricsBuilderConfig(), settings),
```

Comment:
Is this update needed ? 

I'm seeing the other receivers (`nvml`, `mongodb`, `dcgm`) use the `mbConfig.Metrics = s.config.Metrics` to set the metrics configuration, but this one is using the `Default`, so I understand this update creates the same thing as before.

Reply:
Because this used the `DefaultMetricsSettings`, I did a direct mapping to the new way of doing things. I don't actually know why this never used metrics from `Config`. I will raise this to ObservIQ, but that will be another fix PR outside of this one.

Reply:
Thanks! Yeah, now it makes sense.